src/config.rs

```diff
@@ -3631,4 +3629,12 @@ mod tests {
             );
         }
     }
+
+    #[test]
+    fn test_validate_tikv_config() {
+        let mut cfg = TiKvConfig::default();
+        cfg.raft_store.region_split_check_diff = ReadableSize::mb(20);
```

Comment:
How about setting it to `cfg.raft_store.region_split_check_diff + 1`? Thus we can ensure this value is different from the original one.